test/ex_doc/formatter/html_test.exs

```diff
@@ -112,6 +112,24 @@ defmodule ExDoc.Formatter.HTMLTest do
     assert LazyHTML.text(bar_content["h1"]) == "README bar"
   end
 
+  test "extras defined as external urls", %{tmp_dir: tmp_dir} = context do
+    config =
+      doc_config(context,
+        extras: [
+          "readme.md",
```

Comment:
Some OSes have case sensitive file systems, and we normalize everything to lowercase. So I lowercased everything here too, as this was not the concern of this test.